tests_python/my_extensions/pydevd_plugins/extensions/__init__.py

```diff
@@ -0,0 +1,5 @@
+try:
+    __import__('pkg_resources').declare_namespace(__name__)
```

Comment:
pkg_resources resources handles some cases that pkgutil does not.  Most importantly it handles namespace packages bundled inside eggs. While I do not plan to bundle the extension in an egg I think it is a legitimate use case. Hence I prefer trying pkg_resources and if not available using pkgutil.

The form __import__('pkg_resources').declare_namespace(__name__) is the canonical form that setup tools recognized.  It's taken from their docs:
http://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=declare_namespace#namespace-packages